GSoC-2025/ethology ; support for any point trackers (Richard Dushime Mudahera).md

```diff
@@ -0,0 +1,249 @@
+        <strong>Finalization:</strong>- Polish UI/UX.<br>- Finalize tutorials and API docs.
+      </td>
+      <th style="border: 1px solid #ddd; padding: 8px;">Code merged, presentation ready.</th>
+      <td style="border: 1px solid #ddd; padding: 8px;">30</td>
+    </tr>
+    <!-- Week 13 -->
+    <tr>
+      <td style="border: 1px solid #ddd; padding: 8px;">13</td>
+      <td style="border: 1px solid #ddd; padding: 8px;">
+         - Stay active as a community contributor by addressing bugs, offering support, and enhancing features as needed.<br> - Provide assistance through forums, Zulip, and GitHub for users adopting the tool.<br> - Monitor feedback and plan follow-up in agreement with mentors.<br> - Act as a resource for new contributors and maintain a connection with mentors.
+      </td>
+      <th style="border: 1px solid #ddd; padding: 8px;">Continued Community Engagement</th>
+      <td style="border: 1px solid #ddd; padding: 8px;">30</td>
+    </tr>
+  </tbody>
+</table>
+
+### Key Milestones
+
+- Week 3: First end-to-end tracking prototype.
+- Week 6: Mid-term review with core features functional.
+- Week 9: Stretch goals implementation begins.
+- Week 12: Project submission with full test coverage.
+
+### Communication Plan
+- **Mentor Meetings:** Hold weekly video or voice calls to discuss progress, challenges, and next steps.
+- **Daily/Weekly Updates:** Share progress and ask quick questions on Zulip.
+- **Documentation & Issue Tracking:** Use GitHub Issues and project boards to track tasks and milestones.
+- **Code Reviews:** Submit regular or weekly pull requests to ensure the project stays on track and meets community standards.
+
+## Personal Statement
+
+### Past Experience
+I have built a good foundation in Python and computer vision by working on several open source projects. My contributions to NIU repositories show that I can work well with open source communities. While I am still learning machine learning and using PyTorch, I am eager to learn more. I have worked on projects that required both research-level and production-quality code. I have also collaborated with Research Software Engineers and participated in events like the Collaborations Workshop 2024, which has deepened my interest in software sustainability and research projects and tools.
+
+### Motivation: Why This Project?
+Animal behavior research faces challenges because many tools are too strict and do not work well with non-model organisms or new types of behaviors. Any-point tracking is a new approach that can help researchers study animals more easily by tracking any point on the animal in a video, not just fixed body parts. Ethology’s goal of unifying different computer vision tasks under one easy-to-use interface matches my belief that accessible tools can boost scientific discovery. I am excited about this project because it can help researchers ask new questions and gain better insights into animal behavior.
```

Comment:
> Animal behavior research faces challenges because many tools are too strict and do not work well with non-model organisms or new types of behaviors.

Can you provide examples of the tools that are "too strict", or of tools that don't work well with non-model organisms (such as which organisms?) or new types of behaviours?